### DISCUSSION TITLE
Uninstalling Mac Mouse Fix (I read the FAQ)

### DISCUSSION THREAD

**Discussion by workschoolhub** [noah-nuebling/mac-mouse-fix#738] [Questions] [answered]
Hi there - I noticed that the FAQ says to have "Mac Mouse Fix is enabled" and then to move to trash.
"Enabled" must be a typo, right? It should be disabled? I tried enabled, and my mouse setting retained the Mac Mouse Fix's settings even after emptying my trash.

Thank you

**Comment by noah-nuebling** [comment, marked as answer]
Hi, thanks for reaching out! Any effect from Mac Mouse Fix should go away after you disable it. When you move it to the trash, empty the trash, and then restart your computer, I don't think it's possible that there's still any effect from it.

How exactly is your mouse behaving? Maybe I'll have some idea what's going on if you tell me more.

The FAQ says that Mac Mouse Fix should be enabled, because it tries detect when you move it to the trash and then clean up after itself. It can only do this if it's enabled. However, I found out that macOS creates lots of weird files on behalf of the Mac Mouse Fix that I don't know anything about, and I don't know where they all are, so for newer versions of Mac Mouse Fix I gave up on making the app try to clean up after itself. 

I now recommend the awesome [App Cleaner by FreeMacSoft](https://freemacsoft.net/appcleaner/) to clean up any residue that is left behind by Mac Mouse Fix or other apps.

If you have any more questions or concerns, let me know!

**Reply by workschoolhub** [reply]
Thanks very much @noah-nuebling for your reply.

> How exactly is your mouse behaving?

Deleting with Mac Mouse Fix enabled retained its settings (though I didn't try to restart my computer). So what I ended up doing is disabling and then deleting. That did the trick. (I guess disabling without deleting would have produced the same outcome.)

Your explanation for why the FAQ says to keep MMF enabled while uninstalling makes sense to me now. Thanks for clarifying.

Thanks also for the product. I'll keep it in mind for the future. I've been trying to switch from Mos, as Mos bugs up for certain apps. But Mos's customizability in scroll behaviour and also its exception feature is better suited to my needs. 

Many thanks again!
